### PR TITLE
 car_detection in image  OpenCV

### PR DESCRIPTION
**Pull request by ryuk156**
issue :  https://github.com/ankitdobhal/Awesome-Python-Scripts/issues/93

About Pull request : 
This pr consist of car detection in an image if a car is detected in an image the green rectangle will be drawn , it uses pre trained car classifer

Demo : 
image

### CODE REVIEW

**Comment by ryuk156** [comment]
@ankitdobhal review and merge

File: Image-Processing/car_detection_image.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#convert image black and white i.e. grayscale 
+black_and_white=cv2.cvtColor(img,cv2.COLOR_BGR2GRAY)
+
+#create classifier
+car_tracker=cv2.CascadeClassifier(classifier_car)
+
+#detect cars
+cars=car_tracker.detectMultiScale(black_and_white)
+
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py]: Remove new line

File: Image-Processing/car_detection_image.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## the above variable will return 4 cordinates i.e height,width,postionx,positiony
+
+
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py]: remove excess new line

File: Image-Processing/car_detection_image.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+for (x,y,w,h) in cars: 
+    cv2.rectangle(img,(x,y),(x+w,y+h),(0,255,0),3) ##this loop will create grren rectangle when car is detected
+
+
+
+
+
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py]: remove excess new lines

File: Image-Processing/car_detection_image.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+#display image
+cv2.imshow('car image',img)
+
+cv2.waitKey()
+
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py:14]: remove new line at the end

**Review by disha2sinha — CHANGES_REQUESTED** [review]
Don't put so many new lines between codes

**Comment by ryuk156** [comment]
@disha2sinha done review please

File: Image-Processing/car_detection_image.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import cv2
+img_file="car2.jpg"
+classifier_car='car_dect.xml'  #trained car data  Link:"https://gist.github.com/199995/37e1e0af2bf8965e8058a9dfa3285bc6"
+img=cv2.imread(img_file) #create image reader
+black_and_white=cv2.cvtColor(img,cv2.COLOR_BGR2GRAY) #convert image black and white i.e. grayscale 
+car_tracker=cv2.CascadeClassifier(classifier_car) #create classifier
+cars=car_tracker.detectMultiScale(black_and_white) #detect cars
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py]: Put a new line before starting for loop

File: Image-Processing/car_detection_image.py [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+for (x,y,w,h) in cars:  ## the above variable will return 4 cordinates i.e height,width,postionx,positiony
+    cv2.rectangle(img,(x,y),(x+w,y+h),(0,255,0),3) ##this loop will create grren rectangle when car is detected
```
Comment (disha2sinha) [inline Image-Processing/car_detection_image.py:10]: put new line after ending for loop

**Review by disha2sinha — CHANGES_REQUESTED** [review]
Put space around operators and after comma

**Comment by ryuk156** [comment]
@disha2sinha done

**Comment by disha2sinha** [comment]
Add README (guide) along with this inside the folder same as your
Update the README with appropriate installation methods and description guidelines.
Maintain proper Contribution protocols protocols (ignore if you're already following it!) & good practices!

**Pull request closed without merging**